OWNERS.md

```diff
@@ -107,6 +107,17 @@ Scope: The TechDocs plugin and related tooling
 
 These incubating project areas have shared ownership with @backstage/maintainers.
 
+### Notifications
+
+Team: @backstage/notifications-maintainers
+
+Scope: The Notifications and Signals plugins and libraries
+
+| Name        | Organization       | GitHub                                      | Discord      |
+| ----------- | ------------------ | ------------------------------------------- | ------------ |
+| Heikki      | OP Financial Group | [drodil](https://github.com/drodil)         | `deathammer` |
```

Comment:
So got some more information and unfortunately I cannot be part of this even if the organization information is removed.

Reply:
Alright, removed you for now and let's hope it can be worked out still 🤞